src/components/avatars/OcAvatarGroup.vue

```diff
@@ -1,197 +1,37 @@
 <template>
-  <div>
-    <div
-      v-oc-tooltip="tooltip"
-      class="oc-avatar-group"
-      :class="{ 'oc-avatar-group-stacked': stacked }"
-      aria-hidden="true"
-    >
-      <template v-if="avatars.length > 0">
-        <oc-avatar
-          v-for="avatar in avatars"
-          :key="avatar.username"
-          :src="avatar.avatar"
-          :user-name="avatar.displayName"
-          :width="30"
-        />
-      </template>
-      <template v-if="links.length > 0">
-        <oc-avatar-link v-for="(link, index) in links" :key="link.name + index" :name="link.name" />
-      </template>
-      <oc-avatar-count v-if="isOverlapping" :count="users.length - maxDisplayed" />
-    </div>
-    <span class="oc-invisible-sr" v-text="accessibleDescription"></span>
-  </div>
+  <oc-avatar-item icon="group" :name="name" :accessible-label="accessibleLabel"></oc-avatar-item>
 </template>
 
 <script>
-import OcAvatar from "./OcAvatar.vue"
-import OcAvatarCount from "./OcAvatarCount.vue"
-import OcAvatarLink from "./OcAvatarLink.vue"
-
-/**
- * Display a group of avatars
- */
 export default {
   name: "OcAvatarGroup",
   status: "ready",
   release: "2.1.0",
```

Comment:
Somewhat special case because we don't reflect file reuse in here. But I guess since the component is an entirely different one, we should reset the status to `review` and clear the `release` value.